test/test_admin_integration.py

```diff
@@ -80,3 +81,45 @@ def test_create_describe_delete_acls(kafka_admin_client):
 
     assert error is NoError
     assert len(acls) == 0
+
+
+@pytest.mark.skipif(env_kafka_version() < (0, 11), reason="Describe config features require broker >=0.11")
+def test_describe_configs_broker_resource_returns_configs(simple_client, kafka_admin_client):
```

Comment:
Oops, the `simple_client` fixture is going away in #1196... I'm working with @dpkp to figure out how we can get that merged sooner rather than later so we don't waste folks time.

In this case you can actually just get this info via `kafka_admin_client._client._cluster.brokers` so it should be an easy fixup and use the existing fixture.